Documentation/ProjectFile/prj/processes/process/THERMO_MECHANICAL_PHASE_FIELD/t_coupling_scheme.md

```diff
@@ -0,0 +1 @@
+An optional input to select the coupling scheme. The monolithic scheme is currently not verified for THERMO_MECHANICAL_PHASE_FIELD.
```

Comment:
The monolithic scheme must be validated and verified in this PR. Then the second comment can be removed.

Reply:
removed the monolithic scheme.

Reply:
If the monolithic scheme is removed, this is then also obsolete...

Reply:
the coupling_scheme option is removed.